### PR TITLE
Feature/hero components

### PR DESCRIPTION
**Pull request by saurabh-lohiya**

### CODE REVIEW

File: featured/sections/heros/index.ftd [hunk 1]
```diff
@@ -1,58 +1,63 @@
...
-full-width: true
 sidebar: false
 
--- ft-ui.grid-view: Hero with two CTA
-templates: $hero-with-2-cta
+-- ft-ui.grid-view: Hero
+templates: $hero
 more-link-text: View more
-more-link: /featured/sections/heros/hero-with-2-cta/
+more-link: /featured/sections/heros/hero/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:9]:
```suggestion
more-link: /featured/heros/
```

File: featured/sections/heros/index.ftd [hunk 2]
```diff
@@ -1,58 +1,63 @@
...
-more-link: /featured/sections/heros/hero-with-social/
 
--- end: ds.page
 
 
 
+-- ft-ui.template list hero:
 
+-- ft-ui.template: Hero Bottom Hug
+template-url: featured/sections/heros/hero-bottom-hug/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:20]:
```suggestion
template-url: /hero-bottom-hug/
```

File: featured/sections/heros/index.ftd [hunk 3]
```diff
@@ -1,58 +1,63 @@
...
+screenshot: $fastn-assets.files.images.featured.components.hero-bottom-hug.jpg
 
--- ft-ui.template list hero-with-2-cta:
+-- ft-ui.template: Hero Bottom Hug Search
+template-url: featured/sections/heros/hero-bottom-hug-search/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:24]:
```suggestion
template-url: /hero-bottom-hug-search/
```

File: featured/sections/heros/index.ftd [hunk 4]
```diff
@@ -1,58 +1,63 @@
...
+screenshot: $fastn-assets.files.images.featured.components.hero-bottom-hug-search.jpg
 
--- ft-ui.template: Hero with two CTA - Design 5
-template-url: featured/sections/heros/hero-with-2-cta/hero-with-2-cta-5/
-screenshot: $fastn-assets.files.images.featured.sections.heros.hero-with-2-cta.hero-with-two-cta-5.png
+-- ft-ui.template: Hero Left Hug Expanded Search
+template-url: featured/sections/heros/hero-left-hug-expanded-search/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:28]:
```suggestion
template-url: /hero-left-hug-expanded-search/
```

File: featured/sections/heros/index.ftd [hunk 5]
```diff
@@ -1,58 +1,63 @@
...
+screenshot: $fastn-assets.files.images.featured.components.hero-left-hug-expanded-search.jpg
 
--- ft-ui.template: Hero with two CTA - Design 1
-template-url: featured/sections/heros/hero-1/
-screenshot: $fastn-assets.files.images.featured.components.hero-1.png
+-- ft-ui.template: Hero Left Hug Expanded
+template-url: featured/sections/heros/hero-left-hug-expanded/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:32]:
```suggestion
template-url: /hero-left-hug-expanded/
```

File: featured/sections/heros/index.ftd [hunk 6]
```diff
@@ -1,58 +1,63 @@
...
+screenshot: $fastn-assets.files.images.featured.components.hero-left-hug-expanded.jpg
 
--- ft-ui.template: Hero with two CTA - Design 2
-template-url: featured/sections/heros/hero-2/
-screenshot: $fastn-assets.files.images.featured.components.hero-2.png
+-- ft-ui.template: Hero Right Hug Expanded Search
+template-url: featured/sections/heros/hero-right-hug-expanded-search/
```
Comment (gsalunke) [inline featured/sections/heros/index.ftd:36]:
```suggestion
template-url: /hero-right-hug-expanded-search/
```
@saurabh-lohiya change others accordingly.

File: images/featured/components/hero-bottom-hug-search.jpg
Comment (gsalunke) [inline images/featured/components/hero-bottom-hug-search.jpg:1]: @saurabh-lohiya the buttons not looking good. Apply a color scheme to your package and then take screenshots with proper colors applied to all buttons.

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: images/featured/components/hero-bottom-hug-search.jpg
Comment (saurabh-lohiya) [reply images/featured/components/hero-bottom-hug-search.jpg:1]: @gsalunke should I take screenshot of this?
https://saurabh-lohiya.github.io/hero/docs/hero-right-hug-search/
and accordingly

**Pull request merged**